test/acceptance/features/bindAppToServiceWithLabelSelector.feature

```diff
@@ -0,0 +1,111 @@
+Feature: Bind an application with label selector to a service
```

Comment:
should we really consider this as a separate feature? this is all about referring application resources using attached labels

Reply:
I think a separate feature file for the label selector is fine.

Reply:
can you backup your statement with some reason?

Reply:
I am fine with moving to another file. Please suggest the file.